# Position
geometry_msgs/Pose pose

# Description of pose
std_msgs/String description
